sr_port/gtm_env_init.c

```diff
@@ -128,6 +129,11 @@ void	gtm_env_init(void)
 			if (GDL_SmStorHog & tdbglvl)
 				tdbglvl |= GDL_SmBackfill | GDL_SmChkAllocBackfill;
 			ydbDebugLevel |= tdbglvl;
+			ydbSystemMalloc = ((GDL_UseSystemMalloc & ydbDebugLevel) || FALSE);
```

Comment:
I think so too. That it is to make sure the result is a boolean. But none of those seem necessary to me. See a few lines before in the same code snippet. We do an "if (GDL_SmStorHog & tdbglvl)". We are treating the result of the bitwise & as a boolean value. I will fix line 132 to avoid the || FALSE.